FreeRTOS-Plus/VisualStudio_StaticProjects/FreeRTOS-Kernel/FreeRTOSConfig.h

```diff
@@ -139,7 +139,7 @@ void vConfigureTimerForRunTimeStats( void );
  * results in the wired network being used, while setting
  * configNETWORK_INTERFACE_TO_USE to 2 results in the wireless network being
  * used. */
-#define configNETWORK_INTERFACE_TO_USE      ( 1L )
+#define configNETWORK_INTERFACE_TO_USE      ( 4L )
```

Comment:
nit: Should this stay as 1?

Reply:
I think we should move it to 4?
As stated it ends up being the actual Ethernet port on some laptops. Then it was 4 for my virtual windows environment.

Where users will likely need to come in and change this anyways, but maybe this value happens to be correct more often?

Reply:
In this case, I think we should set it to 0. That way the system will print an error with the reason instead of leaving folks to wonder why their demo doesn't work.
See here: https://github.com/FreeRTOS/FreeRTOS/blob/b9cb18e46e7042d1dc1475bf054cc5278e422d16/FreeRTOS-Plus/VisualStudio_StaticProjects/FreeRTOS%2BTCP/NetworkInterface_WinPCap.c#L487

Reply:
> In this case, I think we should set it to 0.
I really like that idea actually, thanks!